src/runmode-af-packet.c

```diff
@@ -286,7 +286,7 @@ static void *ParseAFPConfig(const char *iface)
         }
     }
 
-    if (ConfGetChildValueWithDefault(if_root, if_default, "copy-iface", &out_iface) == 1) {
+    if (ConfGetChildValueStringWithDefault(if_root, if_default, "copy-iface", &out_iface) == 1) {
         if (out_iface != NULL) {
```

Comment:
Is the point of this new function is that we can now skip this check for NULL?

Reply:
oh yes. Updating..